runtime/autoload/man.vim

```diff
@@ -75,7 +75,9 @@ function man#get_page(...) abort
 
   setlocal modifiable
   silent keepjumps norm! 1G"_dG
-  let $MANWIDTH = winwidth(0)
+  if empty($MANWIDTH)
+    let $MANWIDTH = winwidth(0)
+  endif
```

Comment:
@george-b I just noticed that [the ftplugin](https://github.com/noahfrederick/neovim/blob/patch-1/runtime/ftplugin/man.vim#L17-L19) clobbers MANWIDTH. Does it work for you more than once?